.github/workflows/unit_test_coverage.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         run: npm ci
 
       - name: Audit dependencies for security vulnerabilities
-        run: npm audit --production
+        run: npm audit --omit=dev --audit-level=${{ vars.NPM_AUDIT_LEVEL }}
```

Comment:
this can be reverted now if this is related to axios

Reply:
thanks @sb-cecilialiu. Need to update axios dependency as well `"axios": "^1.7.4",` to use vulnerability free version

Reply:
@adongare yeah sure, just updated it.